### DISCUSSION TITLE
[Question] SearXNG's uWSGI app not available

### DISCUSSION THREAD

**Discussion by Nexxol** [searxng/searxng#2959] [Q&A] [answered]
Hi, I'm trying to set up searxng on my raspberry pi but I run into issues using the install script. 

I get

```
Install Apache site searxng.conf
================================

This installs SearXNG's uWSGI app as apache site.  The apache site is
located at: /searxng.conf.

To install uWSGI use::

    searxng.sh install uwsgi

ERROR: ./utils/searxng.sh: line 696: SearXNG's uWSGI app not available
```

and trying to do the searxng.sh install uwsgi comes up with

```
-bash: searxng.sh: command not found
```

**Comment by return42** [comment]
> raspberry pi .. using the install script.

The install scripts are only tested for the Linux distributions you find in the [Step by Step manual](https://docs.searxng.org/admin/installation-apache.html#installation-apache)

```
-bash: searxng.sh: command not found
```

.. I think the message is clear.

**Reply by unixfox** [reply, marked as answer]
Or you can use searxng-docker which support ARM based devices like raspberrypi: https://github.com/searxng/searxng-docker
